docker/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+myriad:
+  build: .
+#    - dockerfile: ResourceManager.dockerfile
```

Comment:
Would you need to uncomment this as you've deleted "Dockerfile" file from last diff, and now only require "ResourceManager.dockerfile" file to build RM docker image?

And, A bit curious to know how docker-compose.yml works here?